inc/html.php

```diff
@@ -233,7 +234,12 @@ function html_btn($name, $id, $akey, $params, $method='get', $tooltip='', $label
         $ret .= 'accesskey="'.$akey.'" ';
     }
     $ret .= 'title="'.$tip.'">';
-    $ret .= hsc($label);
+    if ($inlineSVG) {
+        $ret .= '<span>' . hsc($label) . '</span>';
+        $ret .= $inlineSVG;
```

Comment:
wouldn't it make more sense to pass the path and call inlineSVG() here? Because otherwise the passed parameter isn't really `$inlineSVG` but more like `$whateverHTMLyouWant`

Reply:
Mh, makes sense. Fixed :)